internal/github/checkPendingCI.go

```diff
@@ -21,7 +21,10 @@ type PendingPR struct {
 
 // CheckForPendingCI returns a list of PR URLs that are ready for CI to run but haven't started yet.
 func CheckForPendingCI(ctx context.Context, githubClient *github.Client, cfg *config.Config) ([]PendingPR, error) {
-	teamMembers, _ := GetTeamMemberList(githubClient, cfg.InternalTeam)
+	teamMembers, err := GetTeamMemberList(githubClient, cfg.InternalTeam, cfg.InternalTeamIgnoredUsers)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
No idea why this error wasn't being checked before but... I didn't see a point in not checking it.